go/bitmap_commands_test.go

```diff
@@ -1,15 +1,15 @@
 // Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
 
-package api
+package glide
 
 import (
 	"fmt"
 
-	"github.com/valkey-io/valkey-glide/go/api/options"
+	"github.com/valkey-io/valkey-glide/go/v2/options"
 )
 
-func ExampleGlideClient_SetBit() {
-	var client *GlideClient = getExampleGlideClient() // example helper function
+func ExampleClient_SetBit() {
+	var client *Client = getExampleGlideClient() // example helper function
```

Comment:
I afraid we can't rename client classes, they have to be aligned with other clients

Reply:
Our alignment is not about strict adherence to a naming scheme just for the sake of some nebulous principle when that naming scheme is at odds with idiomatic language conventions. We should be focused on what makes most sense to our users. Feel free to join the discussion https://github.com/valkey-io/valkey-glide/discussions/3808